Tasks/TimeConverter.js

```diff
@@ -0,0 +1,14 @@
+function fromSecondsToMinutes(seconds){
+    return seconds / 60;
+}
+
+function fromSecondsToHours(seconds){
+    return seconds / 3600;
+}
+function fromHoursToMinutes(hours){
+    return hours * 60;
+}
+function fromHoursToSeconds(hours){
+    return hours * 3600;
+}
+module.exports = {fromSecondsToMinutes, fromSecondsToHours, fromHoursToMinutes, fromHoursToSeconds };
```

Comment:
All file and folder names should start with lower case letter `Tasks/TimeConverter.js` ==> `tasks/timeConverter.js`